### ISSUE TITLE
AVKANS Super PTZ Controller

### ISSUE THREAD

**Issue by Waldoweb** [bitfocus/companion-module-requests#1554]
Would love to have an integration for the AVKANS Super PTZ joystick controller to be able to switch the camera being controlled by choosing it on the ATEM Mini preview.

**Comment by NoFear13X** [comment]
I don't think this is possible. I'm trying to do the same thing, but I don't think the joystick can receive commands

![Image](https://github.com/user-attachments/assets/ad3c4a6a-e65d-4af0-b524-8cc54e6e751a)
